salt/_pillar/metalk8s.py

```diff
@@ -49,12 +51,42 @@ def _load_ca(config_data):
     }
 
 
+def _get_endpoints(config="/etc/kubernetes/admin.conf",
+                   context="kubernetes-admin@kubernetes",
+                   namespace="kube-system"):
+    """Find out services endpoints."""
+    endpoints = {}
+
+    if 'kubernetes.show_endpoint' not in __salt__ \
+            or not os.path.exists(config):
+        return endpoints
+
+    for service in ('salt-master', 'registry', 'package-repositories'):
```

Comment:
These strings form a list or a set, not a tuple.

Reply:
will fix